crypto-ffi/bindings/jvm/src/test/kotlin/com/wire/crypto/client/GeneralTest.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+
```

Comment:
I'd omit the while copyright thing. We're not consistent with it anyway and this is already covered by the copyright and other legal notices on both our site and github.

Reply:
Disagree: we have the same notice in all the parallel test files. Omitting it would make this test file the odd one out and might give an external observer the false impression that these general tests are not constrained by the GPLv3 license.

I'd be happy to talk about either enforcing copyright headers through the codebase, or removing them entirely. But that's a separate discussion, outside the scope of this PR.

Reply:
I was not referring to just the test files, but rather to the entire codebase. But ok, let's leave that for another day. :+1: